### ISSUE TITLE
Web help urls in JULES metadata require updating

### ISSUE THREAD

**Issue by maggiehendry** [MetOffice/jules#30]
With the migration of JULES to MetOffice/jules the JULES user guide has also been migrated from [jules-lsm/jules-lsm.github.io](https://github.com/jules-lsm/jules-lsm.github.io). The urls in the metadata therefore point to links that don't exist. These will need updating when the tagged urls exist.

**Comment by james-bruten-mo** [comment]
[Automatic Update]

The Code Review deadline for the Spring 2026 has passed. As this issue does not have a linked pull request it has been removed from the milestone. Please review this issue and either select a new milestone or close it as appropriate. Please contact @MetOffice/ssdteam if you think there has been an error.

 Thanks

**Comment by jennyhickson** [comment]
Adding this back to the milestone as we are trying to get the documentation versioning and links corrected this release.
